lib/erlectricity/port.rb

```diff
@@ -23,7 +23,7 @@ def receive
     def send(term)
       @encoder.out = StringIO.new('', 'w')
       @encoder.write_any(term)
-      data = @encoder.out.string
+      data = @encoder.out.string.force_encoding("ISO-8859-1")
       output.write([data.length].pack("N"))
```

Comment:
There's actually no need to enforce the encoding. You can simply call `data.bytes.length` on line 27, according to the gist [here](https://gist.github.com/sdelano/b753abdb34181baed70e).